src/vue/init.js

```diff
@@ -0,0 +1,24 @@
+import "./logo.js";
+import Vue from "../../node_modules/vue/dist/vue.esm.browser.js";
+import vueI18n from "./i18n.js";
+
+const logoModel = {
+    CONFIGURATOR: CONFIGURATOR,
+    firmwareVersion: "",
+    firmwareId: "",
+    hardwareId: "",
+};
+
+const logoVersionVM = new Vue({
+    el: "#logo",
+    i18n: vueI18n,
+    data: logoModel,
+});
+
+const logoVersionTabsVM = new Vue({
+    el: "#tab_logoversion",
+    i18n: vueI18n,
+    data: logoModel,
+});
+
```

Comment:
Only curiosity, I see two logo entities, one for each of them. Can't be populated all of them in the page with only one call?

Reply:
Nope, you have to be explicit about targeting only single instance. I think rational behind that is to be clear what will modify/depend on the state.